root/relationship/linktype/form.tt

```diff
@@ -1,55 +1,55 @@
 <form action="[% c.req.uri %]" method="post" class="edit-relationship">
     [%- USE r = FormRenderer(form) -%]
 
-    [% form_row_select(r, 'parent_id', add_colon(l('Parent'))) %]
+    [% form_row_select(r, 'parent_id', add_colon('Parent')) %]
 
     [% WRAPPER form_row %]
-        [% r.label('child_order', add_colon(l('Child order'))) %]
+        [% r.label('child_order', add_colon('Child order')) %]
         [% r.number('child_order') %]
         [% field_errors(form, 'child_order') %]
     [% END %]
 
-    [% form_row_text_long(r, 'name', add_colon(l('Name'))) %]
-    [% form_row_text_long(r, 'link_phrase', l('Forward link phrase:')) %]
-    [% form_row_text_long(r, 'reverse_link_phrase', l('Reverse link phrase:')) %]
-    [% form_row_text_long(r, 'long_link_phrase', l('Long link phrase:')) %]
-    [% form_row_checkbox(r, 'is_deprecated', l('This relationship is deprecated')) %]
-    [% form_row_checkbox(r, 'has_dates', l('This relationship allows setting dates')) %]
+    [% form_row_text_long(r, 'name', add_colon('Name')) %]
+    [% form_row_text_long(r, 'link_phrase', 'Forward link phrase:') %]
+    [% form_row_text_long(r, 'reverse_link_phrase', 'Reverse link phrase:') %]
+    [% form_row_text_long(r, 'long_link_phrase', 'Long link phrase:') %]
+    [% form_row_checkbox(r, 'is_deprecated', 'This relationship is deprecated') %]
+    [% form_row_checkbox(r, 'has_dates', 'This relationship allows setting dates') %]
```

Comment:
Same as above

Reply:
The `phrase` strings are already in use elsewhere. For deprecated we have `This relationship type is deprecated and should not be used.` so we probably won't reuse this exact one but that other one. The dates I guess we now show with the dates as attributes, maybe not needed to keep?